src/Backend/Modules/MediaLibrary/Domain/MediaGroup/MediaGroup.php

```diff
@@ -163,6 +163,7 @@ function (MediaGroupMediaItem $connectedItem): MediaItem {
     public function getFirstConnectedMediaItem(): ?MediaItem
     {
         $connectedMediaItems = $this->getConnectedMediaItems();
+        $firstMediaItem = null;
```

Comment:
this commit shouldn't be in here

Reply:
In deed, i noticed this i'm removing this pull request and use the right commits :)